### ISSUE TITLE
fix numpy deprecation warning messages

### ISSUE THREAD

**Issue by bmabey** [bmabey/pyLDAvis#52]

**Comment by themistoklik** [comment]
hey @bmabey ,

you could do a 

```python

   import warnings
   warnings.filterwarnings('ignore')
```

like [this](https://github.com/cemoody/lda2vec/blob/master/examples/twenty_newsgroups/lda.ipynb) guy here?

**Comment by bmabey** [comment]
Haha, that is funny. I was promoted to add this issue when I saw that NB by @cemoody. The `filterwarnings` is a band-aid, but I think it would be best to address the root of the problem and update the code causing the warnings. I haven't spent any time of this though so I may be wrong.

**Comment by themistoklik** [comment]
Yeah that notebook prompted me to let you know. However it would be useful if you put it in the example notebooks as the warnings get pretty nasty and cause major lag/freezes. At least until a permanent fix is issued.
